src/translations/en.js

```diff
@@ -5,6 +5,7 @@ export default {
   FileInputField: {
     browse: 'Browse',
     drop: 'or drop file here',
+    dropHere: 'Drop here!',
```

Comment:
I know that @davidurban  designed it with `Drop here!`, but I would expect `Drop file here` instead to match with translation `Browse or drop file here`. Subject is missing here. 

What do you think @atmelmicro @mbohal?

@davidurban has time off now so we need to decide it without him.

Reply:
I agree with @bedrich-schindler